c++/src/capnp/message.h

```diff
@@ -414,6 +421,21 @@ class FlatMessageBuilder: public MessageBuilder {
   bool allocated;
 };
 
+template <typename MR>
```

Comment:
I see, my original suggestion was that this function accept *any reader type* as input, not a MessageReader.